File: mobile/src/main/java/uk/co/droidinactu/nanowrimo/db/EventLocation.java
package uk.co.droidinactu.nanowrimo.db;

/**
 * Created by aspela on 01/09/16.
 */
public class EventLocation extends AbstractDbObj {

    public final static String COLUMN_NAME_EVENT_ID = "event_id";
    public final static String COLUMN_NAME_LOCATION_ID = "location_id";

    private Event event;
    private Location tag;

    public EventLocation() {
    }

    public EventLocation(Event ebk, Location tg) {
        this.event = ebk;
        this.tag = tg;
    }

    public String toString() {
        return "Book [" + event + "] tag [" + tag + "]";
    }
}

File: mobile/src/main/java/uk/co/droidinactu/nanowrimo/db/EventLocation.kt
package uk.co.droidinactu.nanowrimo.db

/**
 * Created by aspela on 01/09/16.
 */
class EventLocation : AbstractDbObj {

    private lateinit var event: Event
    private lateinit var tag: Location

    constructor() {}

    constructor(ebk: Event, tg: Location) {
        this.event = ebk
        this.tag = tg
    }

    override fun toString(): String {
        return "Book [$event] tag [$tag]"
    }

    companion object {
        val COLUMN_NAME_EVENT_ID = "event_id"
        val COLUMN_NAME_LOCATION_ID = "location_id"
    }
}
